Microsoft.DotNet.Interactive/StreamKernelCommand.cs

```diff
@@ -1,9 +1,19 @@
-﻿namespace Microsoft.DotNet.Interactive
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.DotNet.Interactive
 {
     public class StreamKernelCommand
     {
+        [JsonProperty("id")]
```

Comment:
Do we need both this and the `CamelCasePropertyNamesContractResolver`?

Reply:
the contract resolver is to force the properties of nested objects

Reply:
Understood, but then why are the attributes needed?

Reply:
I prefer to have them if the y get serialized outside of the api. Ideally i'd stick them on the domain commands and events

Reply:
If that happens then the contract on the nested objects would be broken anyway.

For this reason I often make serialization-specific wrapper classes internal. The serialized output is the contract, and the class is an implementation detail that isn't intended to be used for other purposes.